Files/View Models/ItemViewModel.cs

```diff
@@ -848,28 +848,27 @@ private void AddFile(WIN32_FIND_DATA findData, string pathRoot)
                 systemTimeOutput.Second,
                 systemTimeOutput.Milliseconds);
             long fDataFSize = findData.nFileSizeLow;
-            long fileSize;
+            long itemSizeBytes;
             if (fDataFSize < 0 && findData.nFileSizeHigh > 0)
             {
-                fileSize = fDataFSize + 4294967296 + (findData.nFileSizeHigh * 4294967296);
+                itemSizeBytes = fDataFSize + 4294967296 + (findData.nFileSizeHigh * 4294967296);
```

Comment:
What is the significance of the 4294967296 value applied here? Shouldn't we initially store this value as a constant?

Reply:
Of course we can use this value as constant. The main algorithm taken from [here](http://www.freevbcode.com/ShowCode.asp?ID=6836)

Reply:
Good. Let me know when you've made that change

Reply:
@duke7553 Done.